trunk/SUAVE/Methods/Missions/Segments/Single_Point/Fixed_Conditions.py

```diff
@@ -0,0 +1,127 @@
+## @ingroup Methods-Missions-Segments-Single_Point
```

Comment:
Same question as before. Is this mission segment needed?

Reply:
removed